src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -266,6 +267,10 @@ public class SimpleEvents {
 				.description("Called when a player fishes something. This is not of much use yet.")
 				.examples("on fish:")
 				.since("1.0");
+		Skript.registerEvent("Arrow Pickup", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] pickup [an] arrow", "[player] arrow pickup")
+			.description("Called when a player picks up an arrow.")
+			.examples("on pickup an arrow:")
```

Comment:
```suggestion
			.examples("on player arrow pickup:")
```

Reply:
While I prefer the suggested syntax, maybe make player optional?

Reply:
👍

Reply:
> While I prefer the suggested syntax, maybe make player optional?

This is just the example part, in the registered syntax it's already optional